CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 - Convert `NavigationLibrary` to a singleton [#529](http://github.com/openscope/openscope/issues/529)
 - Remove nonfunctional airports from load list broken by feature/809 [#881](http://github.com/openscope/openscope/issues/881)
 - Overhaul of Ronald Reagan Washington National Airport (DC, USA) [#216](https://github.com/openscope/openscope/issues/216)
-
+- Overhaul of Manchester International Airport (England, United Kingdom)[#854](https://github.com/openscope/openscope/issues/854)
```

Comment:
As a general note, try to add changelog entries as a new line, so we have the _addition_ but not the _deletion_. That way as we merge more stuff, we don't run out of room.